### PR TITLE
Fix update for empty queryset

### PR DESCRIPTION
**Pull request by RayeN**
Previously it broke, when we tried to update when no properties were found for given parent object.

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: tests/test_serializers.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def setUp(self):
         self.data = serializer.data
 
     def test_returns_None(self):
```
Comment (stefanor) [inline tests/test_serializers.py]: Needs a rename?

Comment (RayeN) [inline tests/test_serializers.py]: Yeah, missed that.  Thanks.  Fixing.

**Comment by stefanor** [comment]
:+1:

**Pull request merged**